src/components/views/UserPage.js

```diff
@@ -26,6 +26,10 @@ const displayDate = (date) => {
     }
 }
 
+const calculateZoom = (range) => {
+    return Math.log2(21000/range)
+}
+
```

Comment:
why 2100?
but seems to work:)

Reply:
I wanted the circle to be smaller than the map, but bigger than 50% of the map. (between 50% and 100%). With ~40000, the cirle would cover more than the visible map, because the zoomlevel would be one to low. With ~20'000 (half the circumference of the earth), you get at least 50%. I chose 21'000 because I find it okay if the circle touches the boundaries of the map (+ some Spielraum in order not the get a too small circle).